101-mongodb-sharded-on-centos/scripts/config_secondary.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
there is quite a bit of duplication in these scripts as well - should consider combining them - if not, see comments in other scripts and address the same here